### PR TITLE
Fix wrongly compared team

### LINKED ISSUE #68: Wrong team is compared
When updating an application, the team contained in the payload is used for authorization. It should be the team in the database.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #68

### CODE REVIEW

File: src/org/zalando/stups/kio/api.clj [hunk 1]
```diff
@@ -144,8 +144,13 @@
                   :description         nil
                   :specification_type  nil
                   :publicly_accessible false
-                  :criticality_level   2}]
-    (require-write-authorization request (:team_id application))
+                  :criticality_level   2}
+        existing_application (load-application application_id db)]
+
+    (if (nil? existing_application)
```
Comment (harti2006) [inline src/org/zalando/stups/kio/api.clj:150]: can be simplified to `(if existing-application ...`).
Of course "then" and "else" form need to be switched.

Comment (taojang) [inline src/org/zalando/stups/kio/api.clj]: maybe ```(when-not (require-write-authorization ...) (require-write-authorization ...))``` ?

**Comment by taojang** [comment]
:+1:

**Comment by prayerslayer** [comment]
👍

**Pull request merged**